index.html

```diff
@@ -0,0 +1,159 @@
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Join</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">News</a>
+              </li>
+              <li><a href="#" class="contact">Contact us</a></li>
+            </ul>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+          </nav>
+        </div>
+      </section>
+
+      <section class="Section-3">
+        <div class="Content">
+          <h1>
+            2nd Global Submit on Applied Science, Engineering and Technology
+            (GSASET)2023
+          </h1>
+          <div class="infomation">
+            <p>
+              The three days of the conference will contribute to build
+              long-term profitable and sustainable communication among the
+              participants in a wide variety of scientific areas with the common
+              interest in Applied Science, Engineering and Technology.
+              </p>
+            </div>
+            <div class="under-content">
+            </p>
+            <h2>2023.03.23(TUE) ~ 25(WED)</h2>
+            <p>&copy SHAGRI-LA HOTEL, BANGKOK, THAILAND</p>
+          </div>
+          
+        </div>
+      </section>
+    </section>
+
+    
+    <section class="Section-4">
+
+      <h3 class="centering" id="underline">Participation Opportunities
+       
+      </h3>
+
+      <div class="col">
+        <div class="row">
+          <img src="images/listener.png" alt="">
+          <h4>Oral Presentaion</h4>
+          <p>Oral Presentation is an effective way to present your valuable research work to get recognized.</p>
+        </div>
+        <div class="row">
+          <img src="images/poster-presentation.png" alt="">
+          <h4>Oral Presentaion</h4>
+          <p>Oral Presentation is an effective way to present your valuable research work to get recognized.</p>
+        </div>
+        <div class="row">
+          <img src="images/workshop-presentation.png" alt="">
+          <h4>Oral Presentaion</h4>
+          <p>Oral Presentation is an effective way to present your valuable research work to get recognized.</p>
+        </div>
+        <div class="row">
+          <img src="images/poster-presentation.png" alt="">
+          <h4>Oral Presentaion</h4>
+          <p>Oral Presentation is an effective way to present your valuable research work to get recognized.</p>
+        </div>
+        <div class="row">
+          <img src="images/workshop-presentation.png" alt="">
+          <h4>Oral Presentaion</h4>
+          <p>Oral Presentation is an effective way to present your valuable research work to get recognized.</p>
+        </div>
```

Comment:
In conformity with standard best practices and SEO optimization and accessibility purposes, please ensure that alt attribute for all images have valid values.